### DISCUSSION TITLE
Postgres not generate table automatically

### DISCUSSION THREAD

**Discussion by kaiwenxiao** [rubiin/ultimate-nest#354] [Q&A] [answered]
Sir, I am so appreciative of your incredible job, as an newer of Nestjs, For learning aspect, I start an new project aligned with your first version, is hard for me to inspect this weirdo, could you ever take a glance at my problem? Thanks a lot

**Comment by kaiwenxiao** [comment]
![7dc476b9709cbd748e1009678111fb8](https://github.com/rubiin/ultimate-nest/assets/31737119/8552a5af-04b3-4862-8fd3-edcba48874c0)

![714a72f06014f4c797905e0511134c0](https://github.com/rubiin/ultimate-nest/assets/31737119/0627c4b7-b294-4ae0-be21-c7081b210af3)

![c82ab008b24be1bf3814e880efb898b](https://github.com/rubiin/ultimate-nest/assets/31737119/3b70195c-71f6-4dcd-a12d-29b3100d0e98)

![f3ad5a293c40f10c93ea0be4f841a0f](https://github.com/rubiin/ultimate-nest/assets/31737119/14a8703c-6c9e-4f56-a222-4ef219ac5f90)


Some Info for refer, I'm using Postgres16 and Navicat(visualiztion for SQL Tool). Could you help me? Thanks, if more info need to provide for investigating, please message me freely. Thanks.

**Comment by rubiin** [comment, marked as answer]
are you using npm scripts or justfile. I think you forgot to migrate the db. This will persist the migrations on our db.
https://github.com/rubiin/ultimate-nest#migration-and-seeding did you run this?

**Reply by kaiwenxiao** [reply]
> are you using npm scripts or justfile. I think you forgot to migrate the db. This will persist the migrations on our db.
> 
> https://github.com/rubiin/ultimate-nest#migration-and-seeding did you run this?

Thanks, let me have a try!
